packages/babel-plugin-relay/BabelPluginRelay.js

```diff
@@ -17,6 +17,12 @@ const getValidGraphQLTag = require('./getValidGraphQLTag');
 const getValidRelayQLTag = require('./getValidRelayQLTag');
 const invariant = require('./invariant');
 
+let RelayConfig;
+try {
+  // eslint-disable-next-line no-eval
+  RelayConfig = eval('require')('relay-config');
+} catch (_) {}
```

Comment:
hmm - why suppress this error?

Reply:
Because we still want the plugin to be able to load its configuration from `.babelrc` if the user doesn’t use a single configuration file.